hack/ginkgo-e2e.sh

```diff
@@ -81,7 +81,10 @@ if [[ -n "${CONFORMANCE_TEST_SKIP_REGEX:-}" ]]; then
   ginkgo_args+=("--skip=${CONFORMANCE_TEST_SKIP_REGEX}")
   ginkgo_args+=("--seed=1436380640")
 fi
-if [[ ${GINKGO_PARALLEL} =~ ^[yY]$ ]]; then
+if [[ -n "${GINKGO_PARALLEL_NODES:-}" ]]; then
+  ginkgo_args+=("--nodes=${GINKGO_PARALLEL_NODES}")
```

Comment:
I wonder if instead of creating this new variable we should just come up with a reasonable default and use that when GINKGO_PARALLEL is set, rather than using -p?